### ISSUE TITLE
Nice Job! Question about batch size?

### ISSUE THREAD

**Issue by cyysc1998** [MS-Diffusion/MS-Diffusion#5]
Thanks for the excellent work! I have a question about the training. In my opinion, each image is composed by different number of subject. So the dimension of the input of Grounding Resampler will be different. How to combine them into a batch?  may be Padding? Thank you!

**Comment by sherrydoge** [comment]
Yes, we set 4 subjects for each example. For those examples not equal to 4 subjects, they will be either discarded or padded to 4. Since we use attention masks in the cross-attentions, the padding parts will be neglected. Details can be found in the Sec. A and Sec. 3.4 of the paper.

**Comment by cyysc1998** [comment]
Ok. Thanks for your reply!
